services-geojson/src/main/java/com/mapbox/geojson/shifter/CoordinateShifterManager.java

```diff
@@ -58,4 +58,12 @@ public static void setCoordinateShifter(CoordinateShifter coordinateShifter) {
     CoordinateShifterManager.coordinateShifter =
       coordinateShifter == null ? DEFAULT : coordinateShifter;
   }
+
+  /**
+   * Check whether the current shifter is the default one.
+   * @return true if using default shifter.
+   */
+  public static boolean isUsingDefaultShifter() {
+    return coordinateShifter == DEFAULT;
```

Comment:
this should be `equals` instead of `==`

Reply:
It seems use `==` doesn't have any issue here as `coordinateShifter` will be the same object of  `DEFAULT` when using default shifter. Do you have any other reasons for using `equals` here?